includes/options/class-amp-options-menu.php

```diff
@@ -140,8 +140,8 @@ public function add_menu_items() {
 	public function render_theme_support() {
 		$theme_support = AMP_Options_Manager::get_option( 'theme_support' );
 
-		$native_description       = __( 'Integrates AMP as the framework for your site by using the active’s theme templates and styles to render AMP responses. This means your site is <b>AMP-first</b> and your canonical URLs are AMP! Depending on your theme/plugins, a varying level of development work may be required.', 'amp' );
-		$transitional_description = __( 'Uses the active theme’s templates to generate non-AMP and AMP versions of your content, allowing for each canonical URL to have a corresponding (paired) AMP URL. This mode is useful to progressively transition towards a fully AMP-first site. Depending on your theme/plugins, a varying level of development work may be required.', 'amp' );
+		$native_description       = __( 'Integrates AMP as the framework for your site by using the active’s theme templates and styles to render AMP responses. This means your site is <b>AMP-first</b> and your canonical URLs are AMP! Depending on your theme/plugins, a varying level of <a href="https://amp-wp.org/documentation/developing-wordpress-amp-sites/">development work</a> may be required.', 'amp' );
+		$transitional_description = __( 'Uses the active theme’s templates to generate non-AMP and AMP versions of your content, allowing for each canonical URL to have a corresponding (paired) AMP URL. This mode is useful to progressively transition towards a fully AMP-first site. Depending on your theme/plugins, a varying level of <a href="https://amp-wp.org/documentation/developing-wordpress-amp-sites/">development work</a> may be required.', 'amp' );
```

Comment:
@swissspidy Should the URL be a string format placeholder since translators should not be changing it, as we don't have a localized site?

Reply:
Oops, I see you commented as such in https://github.com/ampproject/amp-wp/pull/2116#issuecomment-482507052